lib/faraday/options.rb

```diff
@@ -252,7 +252,8 @@ def new_builder(block)
   end
 
   class Env < Options.new(:method, :body, :url, :request, :request_headers,
-    :ssl, :parallel_manager, :params, :response, :response_headers, :status)
+    :ssl, :parallel_manager, :params, :response, :response_headers, :status,
+    :reason_phrase)
```

Comment:
I'm not ecstatic about the word `reason_phrase`. Could we use `status_text` instead?